src/bluesearch/mining/eval.py

```diff
@@ -143,11 +144,8 @@ def spacy2df(
         if isinstance(
             pipe,
             (
-                spacy.pipeline.EntityRuler,
-                spacy.pipeline.EntityRecognizer,
-                spacy.pipeline.Lemmatizer,
-                spacy.pipeline.AttributeRuler,
-                spacy.pipeline.Tagger,
+                spacy.pipeline.ner.EntityRecognizer,
+                spacy_transformers.pipeline_component.Transformer,
             ),
         ):
```

Comment:
Like Francesco mentioned in the zoom - wouldn't this change break the eval script for other models?

Reply:
As mentioned in the PR description, this is a patch and this is only for this experiment. Having a `or` to have this part of the code pass is planned.

Reply:
a3965e79f1a3f489b25fa5209498823ff53ab1e6 moves from a patch to a complete integration.

Tests pass and evaluation works for all models.